src/app/components/lineage-graph/lineage-graph.component.ts

```diff
@@ -38,6 +41,7 @@ export class LineageGraphComponent implements OnChanges, OnInit {
     public showMiniMap = true;
     public miniMapPosition: MiniMapPosition;
     public graphNodes: Node[];
+    public datasetKind: string;
```

Comment:
enum.
If you need this to capitalize the letter, I guess it should be done as a pipe

Reply:
Found pipe from Angular ==> titlecase